guest/score/score_exports.go

```diff
@@ -0,0 +1,49 @@
+//go:build tinygo.wasm
+
+/*
+   Copyright 2023 The Kubernetes Authors.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package score
+
+import (
+	"sigs.k8s.io/kube-scheduler-wasm-extension/guest/api"
+	"sigs.k8s.io/kube-scheduler-wasm-extension/guest/internal/imports"
+	"sigs.k8s.io/kube-scheduler-wasm-extension/guest/internal/types"
+)
+
+// score is only exported to the host.
+//
+//export score
+func score() uint64 {
+	// Zero on unconfigured score
+	if Plugin == nil {
```

Comment:
actually a little different. So this is the guest side. Anyone importing this package will export the function, so the host will think it should be run. I think this could be changed to a panic because if it is still nil, it means someone imported the package, but didn't assign a function to it.

Reply:
ok switched it to panic because not setting this is a bug.